hydromt_wflow/workflows/river.py

```diff
@@ -31,6 +31,8 @@ def river(
     river_upa=30.0,
     slope_len=2e3,
     min_rivlen_ratio=0.1,
+    min_rivlen=None,
```

Comment:
Should we not combine min_rivlen_ratio and min_rivlen?

Reply:
Yes, as discussed we now use only `min_rivlen_ratio`.